lib/java_buildpack/container/tomee/tomee_instance.rb

```diff
@@ -34,6 +44,21 @@ def tomcat_7_compatible
         @version < TOMEE_7
       end
 
+      private
+
+      def web_inf_lib
+        if ear?
+          # copy additional libraries to tomee lib folder
+          tomcat_lib
```

Comment:
If we are okay with the new set of changes I think we have satisfied all the requirements and I can add in a little more documentation and we should be good.